esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6019,7 +6019,7 @@ SecAccessFlags translateToSecAccessFlags(CSecAccessType from)
     }
 }
 
-void CWsDfuEx::dFUFileAccessCommon(IEspContext &context, const CDfsLogicalFileName &lfn, const char *requestId, unsigned expirySecs, bool returnTextResponse, IEspDFUFileAccessInfo &accessInfo)
+void CWsDfuEx::dFUFileAccessCommon(IEspContext &context, const CDfsLogicalFileName &lfn, SessionId clientSessionId, const char *requestId, unsigned expirySecs, bool returnTextResponse, unsigned lockTimeoutMs, IEspDFUFileAccessResponse &resp)
```

Comment:
is the lockTimeoutMs param actually used ?

Reply:
appears not, should be passed to lookup (as it is in publish), will fix.